bin/fin

```diff
@@ -2542,9 +2556,91 @@ project_create ()
 	fin init
 }
 
+# Resets Docksal system services
+system_reset ()
+{
+	# TODO: add /etc/exports, etc. here for a complete reset of Docksal
+
+	# Configure network has to happen before any communications with the docker daemon happen (before check_docker_running)
+	if [[ "$1" == "" ]] ; then
+		echo-green 'Resetting network settings...'
+		configure_network
+		echo-green 'Resetting Docksal services...'
+		echo ' * proxy'
+		install_proxy_service
+		echo ' * dns and resolver for .docksal domain'
+		install_dns_service
+		configure_resolver
+		echo ' * ssh-agent'
+		install_sshagent_service
+		return
+	fi
+
+	# Configure network has to happen before any communications with the docker daemon happen (before check_docker_running)
+	if [[ "$1" == "network" ]] ; then
+		echo-green 'Resetting network settings...'
+		configure_network
+		return
+	fi
+
+	if [[ "$1" == "vhost-proxy" ]] ; then
+		echo-green 'Resetting Docksal HTTP/HTTPS reverse proxy service...'
+		install_proxy_service
+		return
+	fi
+
+	if [[ "$1" == "dns" ]] ; then
+		echo-green 'Resetting Docksal DNS service and configuring resolver for .docksal domain...'
+		install_dns_service
+		configure_resolver
+		return
+	fi
+
+	if [[ "$1" == "ssh-agent" ]] ; then
+		echo-green 'Resetting Docksal ssh-agent service...'
+		install_sshagent_service
+		return
+	fi
+}
+
+# Disaplys Docksal system status
+system_status ()
+{
+	# TODO: replace with a custom view showing services, networks, vm, etc. status
+	docker ps --filter "label=io.docksal.group=system"
+}
+
+# Starts all Docksal related things
+# TODO: add projects, network, /etc/exports, etc. here for a complete shutdown of Docksal
+system_start ()
+{
+	if ! is_linux && ! is_docker_native && ! is_wsl; then
+		echo "Starting Docksal VM..."
+		vm start
+	else
+		echo "Starting Docksal system services..."
+		system_reset
+	fi
+}
+
+# Stops/disables all Docksal things
+# TODO: add projects, network, /etc/exports, etc. here for a complete shutdown of Docksal
+system_stop ()
+{
+	if ! is_linux && ! is_docker_native && ! is_wsl; then
+		echo-green "Stopping Docksal VM..."
```

Comment:
Shouldn't configure_resolver off be here as well?

Reply:
Yes, however it's already present in `vm stop`